### ISSUE TITLE
Store `ledger_hash` with validators

### ISSUE THREAD

**Issue by ckniffen** [ripple/validator-history-service#67]
Right now we store the last ledger index we have validations for.  We should additionally store the `ledger_hash`.  This is important because it can be used to see if it is out of sync.

This will also clean up the validators page in the explorer which doesn't apply ledger hash color until the next ledger is validated.

**Comment by ckeshava** [comment]
I personally prefer the `ledger_index` values to check if a node is out-of-sync. Isn't it hard to read the `ledger_hash` values?

**Comment by ckeshava** [comment]
> Why not both? The Explorer needs both.

I understand, I'm trying to think of use cases where humans use `ledger_hash` values.

**Comment by mvadari** [comment]
> > Why not both? The Explorer needs both.
> 
> I understand, I'm trying to think of use cases where humans use `ledger_hash` values.

Different networks may be on roughly the same `ledger_index` value, but they'll have different `ledger_hash` values. e.g. if Testnet and Devnet are restarted at around the same time.

**Comment by ckeshava** [comment]
> Different networks may be on roughly the same ledger_index value, but they'll have different ledger_hash values. e.g. if Testnet and Devnet are restarted at around the same time.

That is true. However, on what occasions will developers see validated ledgers from different networks? Does this happen in practice?

The examples that I can think of -- WebSocket `ledger` streams, JSON-RPC requests of the `ledger` command etc are targeting one specific node. I'm guessing that users are aware of the `ValidatorList` used by such nodes (hence their "network" is well-known to the public).

**Comment by mvadari** [comment]
It happens sometimes when a node is switching networks and still has peers from the old network.
